pages/dug_10/4-osfv-status.md

```diff
@@ -0,0 +1,374 @@
+---
+theme: /slidev-template/theme
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+## Dasharo Open Source Firmware Validation Status
+
+<center><img src="/img/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px"></center>
+
+---
+
+# <center> Agenda </center>
+
+* Short introduction to OSFV
+* Stats
+* Improvements
+* Work in progress & future plans
+* Q&A
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# <center> Introduction to Open Source Firmware Validation </center>
+
+---
+
+# <center> Introduction to Open Source Firmware Validation </center>
+
+- Open Source Validation of Open Source Firmware
+- Based on Robot Framework
+- Allows us to perform hundreds of automated tests
+  to ensure the quality of Dasharo
+
+For more details check the previous OSFV Status presentations at
+DUG #08 and DUG #06 at [vpub.dasharo.com](https:/vpub.dasharo.com).
```

Comment:
According to discussions you see in the internal chat, this is the old system which is going to be closed. In such a case, we should avoid generating new references to it.
Maybe instead we could link to schedule of previous events:
- https://cfp.3mdeb.com/developers-vpub-0xd-2024/schedule/
- https://cfp.3mdeb.com/developers-vpub-0xb-2024/schedule/